Client/game_sa/gamesa_init.h

```diff
@@ -104,3 +104,45 @@ void MemOrFast(U ptr, const T value)
 
 bool GetDebugIdEnabled(uint uiDebugId);
 void LogEvent(uint uiDebugId, const char* szType, const char* szContext, const char* szBody, uint uiAddReportLogId = 0);
+
+struct __THISCALL{};
+struct __CDECL{};
+struct __STDCALL{};
+struct __FASTCALL{};
+struct __VECTORCALL{};
```

Comment:
`typedef` ?

Reply:
What for?

Reply:
Structures are empty, they have no data or methods, so typedef is completely unnecessary here imo

Reply:
so `using __CDECL = struct{};`?

Reply:
nvm, it won't work